docs/streams/upgrade-guide.html

```diff
@@ -136,9 +136,16 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
     <h3><a id="streams_api_changes_370" href="#streams_api_changes_370">Streams API changes in 3.7.0</a></h3>
     <p>
         IQv2 supports <code>RangeQuery</code> that allows to specify unbounded, bounded, or half-open key-ranges, which return data in ascending (byte[]-lexicographical) order (per partition).
-        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-985%3A+Add+reverseRange+and+reverseAll+query+over+kv-store+in+IQv2">KIP-985</a> extends this functionality by adding <code>.withDescendingKeys()<code> to allow user to receive data in descending order.
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-985%3A+Add+reverseRange+and+reverseAll+query+over+kv-store+in+IQv2">KIP-985</a> extends this functionality by adding <code>.withDescendingKeys()</code> to allow user to receive data in descending order.
     </p>
 
+    <P>
+        IQv2 supports <code>TimestampedKeyQuery</code> that allows to retrieve a single record(ValueAndTimestamp) based on its key.
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-992%3A+Proposal+to+introduce+IQv2+Query+Types%3A+TimestampedKeyQuery+and+TimestampedRangeQuery">KIP-992</a>
+        IQv2 supports <code>TimestampedRangeQuery</code> that allows to retrieve a set of records(ValueAndTimestamp), specified using an upper and/or lower bound on the keys.
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-992%3A+Proposal+to+introduce+IQv2+Query+Types%3A+TimestampedKeyQuery+and+TimestampedRangeQuery">KIP-992</a>
```

Comment:
I think we need to reword this, because `Timestamped[Key/Range]Query` are new.

```
KIP-992 add two new query types, namely TimestampedKeyQuery and TimestampedRangeQuery. Both should be used to query a timestamped key-value store, to retrieve a ValueAndTimestampe result. The existing KeyQuery and RangeQuery are changes to always return the value only, also for timestamped key-value stores.
```

Of course, please add the proper markup for the KIP link and `<code>` styles.

Reply:
ok, fixed it.